src/client/pfs_file.go

```diff
@@ -258,6 +236,14 @@ func (mfc *modifyFileCore) sendDeleteFile(req *pfs.DeleteFile) error {
 	})
 }
 
+// Close closes the ModifyFileClient.
+func (mfc *modifyFileClient) Close() error {
```

Comment:
Can this be called more than once; what are the semantics here?

Reply:
The same as the underlying gRPC stream. Should we support calling this more than once?

Reply:
I guess the underlying semantics are fine.  It's not going to panic.  I want to be able to defer Close it without having the multiple close problems that we had in the storage layer.

Reply:
I think it is reasonable for multiple closes to be a client error, and for failing the call the context should be canceled without a close call. That lines up with the gRPC stream behavior.